karapace/rapu.py

```diff
@@ -333,9 +335,9 @@ async def _handle_request(
                 headers = ex.headers
             except:  # pylint: disable=bare-except
                 self.log.exception("Internal server error")
+                headers = {"Content-Type": "application/json"}
                 data = {"error_code": HTTPStatus.INTERNAL_SERVER_ERROR.value, "message": "Internal server error"}
                 status = HTTPStatus.INTERNAL_SERVER_ERROR
-                headers = {}
```

Comment:
The lack of headers IMO is a bug, and that was cluttering the failures in the tests because `requests.get().json()` throws an exception because the response was lacking the correct content type.